bounties/npm/strip-ansi/1/vulnerability.json

```diff
@@ -0,0 +1,57 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2021-01-09",
+    "AffectedVersionRange": "*",
+    "Summary": "Prototype Pollution",
+    "Contributor": {
+        "Discloser": "tibinsunny",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "strip-ansi",
+        "URL": "https://www.npmjs.com/package/strip-ansi",
+        "Downloads": "61107490"
+    },
+    "CWEs": [
+        {
+            "ID": "471",
+            "Description": "Modification of Assumed-Immutable Data (MAID)"
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "L",
+        "PR": "N",
+        "UI": "N",
+        "S": "U",
+        "C": "L",
+        "I": "L",
+        "A": "L",
+        "E": "",
```

Comment:
Please add `"P"` here as you have attached a POC.

Once the checks run again, this should automatically merge your PR.